### PR TITLE
feat(PT-13110): Add plugin workflow

### PR DESCRIPTION
**Pull request by rathmerdominik**

### CODE REVIEW

**Comment by rathmerdominik** [comment]
Do not forget to squash and merge! :smile:

File: .github/workflows/call-plugin-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    push:
+        branches:
+            - master
+
+jobs:
+    call-test-workflow:
+        strategy:
+            matrix:
+                include:
+                    - shopware-version: 'v5.6'
```
Comment (mitelg) [inline .github/workflows/call-plugin-workflow.yml]: correct tag? see SwagEssentials

File: .github/workflows/call-plugin-workflow.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                      mysql-version: '8.0'
+
+        uses: shopware5/docker-images-testing/.github/workflows/php-unit-tests-shopware.yml@main
+        with:
+            plugin-name: SwagUserPrice
+            php-version: ${{ matrix.php-version }}
+            shopware-version: ${{ matrix.shopware-version }}
+            mysql-version: ${{ matrix.mysql-version }}
+
+    call-analyse-workflow:
```
Comment (mitelg) [inline .github/workflows/call-plugin-workflow.yml:33]: javascript analysis is missing

**Review by mitelg — CHANGES_REQUESTED** [review]
why are they not running now?

File: .github/workflows/call-plugin-workflow.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        name: Analyse code for SwagUserPrice
+        uses: shopware5/docker-images-testing/.github/workflows/php-code-analysis.yml@main
+        with:
+            plugin-name: SwagUserPrice
+
+    javascript-code-analysis:
```
Comment (mitelg) [inline .github/workflows/call-plugin-workflow.yml]: please add this plugin to this ticket https://shopware.atlassian.net/browse/PT-13120

would be nice if you could also copy the code there. we should make that a reusable workflow, too

**Review by mitelg — CHANGES_REQUESTED** [review]

File: .github/workflows/call-plugin-workflow.yml (hunk 3 above)
Comment (rathmerdominik) [reply .github/workflows/call-plugin-workflow.yml]: Done and deleted :+1:

**Review by mitelg — APPROVED** [review]

**Pull request merged**